specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/examples/WorkspacesAvailableServiceTiers.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "resourceGroupName": "rg1",
+    "workspaceName": "workspace1",
+    "api-version": "2020-03-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-00000000000"
+  },
+  "responses": {
+    "200": {
+      "body": [
+        {
+          "ServiceTier": "PerNode",
```

Comment:
Properties are case-sensitive and don't align with definition in swagger.

Reply:
fixed